core/sql/comexe/ComTdbExeUtil.cpp

```diff
@@ -2417,6 +2425,15 @@ short ComTdbExeUtilLobShowddl::getLOBnum(short i)
   return lobNum;
 }
 
+NABoolean ComTdbExeUtilLobShowddl::getIsExternalLobCol(short i)
+{
+  if ((i > numLOBs_) || (i <= 0))
+    return -1;
```

Comment:
Removing the check since the calling code (ExeUtilGet.cpp) checks for the input being between 1and max num of lobs